packages/fast-tooling/package.json

```diff
@@ -74,7 +74,8 @@
         "typescript": "^3.3.3"
     },
     "peerDependencies": {
-        "lodash-es": "^4.0.0"
+        "lodash-es": "^4.0.0",
+        "ajv": "^6.10.0"
```

Comment:
So, I think making ajv a dependency here is actually an issue, I'm considering adding another package just for this utility called `@microsoft/fast-tooling-ajv-validator` what do people think?

Reply:
What is the concern?

Reply:
That we have an additional dependency added to the package. We're arbitrarily deciding to use `ajv` as the validator here, but since this is a stand-alone utility, where we are inviting others to create their own. It might make more sense to have this exist as part of another package, but it could also be tree shaken.

Reply:
I've moved it from peer to dependency.

Reply:
That will create the opportunity for users to tree-shake if they decide to use it and be less cumbersome.